### PR TITLE
Fixes #9770: Document the usecases for audit/enforce modes

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9770

### CODE REVIEW

File: 3_usecases/0_usecases_intro.txt [hunk 1]
```diff
@@ -30,3 +30,19 @@ servers. SSH servers can then be stopped or reconfigured manually many times,
 Rudder will always restore your preferred settings and restart the SSH server in
 less than 5 minutes.
 
+=== Using Rudder as an Audit tool
+
+Using Rudder as an Audit tool is useful if you do not want to make anychanges or the system,
```
Comment (jooooooon) [inline 3_usecases/0_usecases_intro.txt]: s/anychanges/any changes/

Comment (jooooooon) [inline 3_usecases/0_usecases_intro.txt]: s/or/on/

File: 3_usecases/0_usecases_intro.txt [hunk 2]
```diff
@@ -30,3 +30,19 @@ servers. SSH servers can then be stopped or reconfigured manually many times,
 Rudder will always restore your preferred settings and restart the SSH server in
 less than 5 minutes.
 
+=== Using Rudder as an Audit tool
+
+Using Rudder as an Audit tool is useful if you do not want to make anychanges or the system, 
+temporary (freeze period, etc.) or permanently.
```
Comment (jooooooon) [inline 3_usecases/0_usecases_intro.txt]: s/temporary/temporarily/

**Comment by amousset** [comment]
Commit modified

**Pull request merged**